pkg/cmd/gen_driftignore.go

```diff
@@ -16,19 +17,34 @@ func NewGenDriftIgnoreCmd() *cobra.Command {
 	cmd := &cobra.Command{
 		Use:   "gen-driftignore",
 		Short: "Generate a .driftignore file based on your scan result",
-		Long:  "This command will generate a new .driftignore file containing your current drifts and send output to /dev/stdout\n\nExample: driftctl scan -o json://stdout | driftctl gen-driftignore -i /dev/stdin > .driftignore",
+		Long:  "This command will generate a new .driftignore file containing your current drifts\n\nExample: driftctl scan -o json://stdout | driftctl gen-driftignore",
 		Args:  cobra.NoArgs,
 		RunE: func(cmd *cobra.Command, args []string) error {
-			if opts.InputPath == "" {
-				return errors.New("Error: you must specify an input to parse JSON from. Use driftctl gen-driftignore -i <drifts.json>\nGenerate a JSON file using the output flag: driftctl scan -o json://path/to/drifts.json")
+			driftFile := os.Stdin
+			if opts.InputPath != "-" {
+				var err error
+				driftFile, err = os.Open(opts.InputPath)
+				if err != nil {
+					return err
+				}
+				defer driftFile.Close()
 			}
 
-			_, list, err := genDriftIgnore(opts)
+			_, list, err := genDriftIgnore(opts, driftFile)
 			if err != nil {
 				return err
 			}
 
-			fmt.Println(list)
+			ignoreFile := os.Stdout
+			if opts.OutputPath != "-" {
+				var err error
+				ignoreFile, err = os.OpenFile(opts.OutputPath, os.O_CREATE|os.O_TRUNC|os.O_WRONLY, 0644)
+				if err != nil {
+					return errors.Wrapf(err, "can't open file %s", opts.OutputPath)
```

Comment:
We are hiding the root cause of why the file can't be create here, maybe we should just keep the original error since `os.Openfile` errors (`open /that/does/not/exist: no such file or directory
`) are explicit enough WDYT ?